pages/collection.tsx

```diff
@@ -11,28 +13,43 @@ import {
   PPagination,
   PSpinner,
 } from '@porsche-design-system/components-react';
-import React, { useState } from 'react';
+import type { AccordionChangeEvent } from '@porsche-design-system/components-react';
+import React, { useCallback, useState } from 'react';
 import Header from '../components/header';
 import Head from 'next/head';
 
 const CollectionPage = (): JSX.Element => {
   const [submit, setSubmit] = useState(false);
   const [activePage, setActivePage] = useState(1);
+  const [isAccordion1Open, setIsAccordion1Open] = useState<boolean>(false);
```

Comment:
boolean generic could be removed here

Reply:
thats exactly the way we recommend it on our storefront

Reply:
There we also talked about removing it in the PR, but it was not worth the effort :D